BUILDING.md

```diff
@@ -100,6 +100,21 @@ To install this version of Node.js into a system directory:
 $ [sudo] make install
 ```
 
+#### Using [nvm](https://github.com/creationix/nvm)
+You can use a Node.js version built from source by installing into the location
+that nvm expects:
+
+```console
+$ make install DESTDIR='$(HOME)/.nvm/versions/node/' PREFIX='$(FULLVERSION)-pre'
```

Comment:
instead of hardcoding the versions path, it'd be ideal to use `nvm_version_path "$VERSION"`